src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -44,6 +44,7 @@ class Loader {
     this._shouldAutoMock = true;
     this._extensions = config.moduleFileExtensions.map(ext => '.' + ext);
 
+    this._virtualModules = {};
```

Comment:
This should be `Object.create(null)` so it doesn't create an Object with a prototype and we can do property lookups without `hasOwnProperty`.